sct.py

```diff
@@ -981,6 +985,21 @@ def collect_logs(test_id=None, logdir=None, backend=None, config_file=None):
 
     click.echo(table.get_string(title="Collected logs by test-id: {}".format(collector.test_id)))
 
+    try:
+        test_run = ArgusTestRun.get(test_id=UUID(collector.test_id))
+        if test_run:
+            for cluster_type, s3_links in collected_logs.items():
+                for link in s3_links:
+                    if isinstance(link, str):
+                        test_run.run_info.logs.add_log(cluster_type, link)
+                    if isinstance(link, list):
+                        for sub_link in link:
+                            test_run.run_info.logs.add_log(cluster_type, sub_link)
```

Comment:
think of how to put this logic in one method/function

Reply:
Done

Reply:
done